lib/ts/recipe/thirdparty/recipe.ts

```diff
@@ -92,6 +92,7 @@ export default class Recipe extends RecipeModule {
                         emailDelivery: undefined,
```

Comment:
i had removed the use of thirdparty as a secondary factor cause of the following reason: https://supertokens.slack.com/archives/C051BN9QJUX/p1705166104069829